code/game/atoms_movable.dm

```diff
@@ -506,15 +506,15 @@
 // Here's where we rewrite how byond handles movement except slightly different
 // To be removed on step_ conversion
 // All this work to prevent a second bump
-/atom/movable/Move(atom/newloc, direction, glide_size_override = 0)
+/atom/movable/Move(atom/newloc, direction, glide_size_override = 0, update_dir = TRUE)
 	. = FALSE
 	if(!newloc || newloc == loc)
 		return
 
 	if(!direction)
 		direction = get_dir(src, newloc)
 
-	if(set_dir_on_move && dir != direction)
+	if(set_dir_on_move && dir != direction && update_dir)
```

Comment:
Bad? I added this to the Move() proc so I could move peeps without the constant dirchanges

https://user-images.githubusercontent.com/7501474/208247828-843d9d6d-4d53-4e51-ad4a-8d22090c0e23.mp4

I can also just change the variable for the duration of the move code, but doing so would be kinda shitty